src/main/scala/cromwell/binding/types/WdlAnyType.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding.types
+
+case object WdlAnyType extends WdlType {
+  val toWdlString: String = s"Any"
+
+  override protected def coercion = {
+    case any: Any =>
```

Comment:
Can you explain (ultimate via comments in code) why you are matching on `Any`? I'm sure it makes sense to do so but it's not obvious.